modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/ddl/CreateTableCommand.java

```diff
@@ -18,284 +18,125 @@
 package org.apache.ignite.internal.processors.query.calcite.prepare.ddl;
 
 import java.util.List;
+import java.util.Set;
+import org.apache.ignite.schema.definition.ColumnDefinition;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * CREATE TABLE statement.
  */
-public class CreateTableCommand implements DdlCommand {
-    /**
-     * Schema name upon which this statement has been issued - <b>not</b> the name of the schema where this new table will be created.
-     */
-    private String schemaName;
-
-    /** Table name. */
-    private String tblName;
-
-    /** Cache name upon which new cache configuration for this table must be based. */
-    private String templateName;
-
-    /** Name of new cache associated with this table. */
-    private String cacheName;
-
-    /** Name of cache key type. */
-    private String keyTypeName;
-
-    /** Name of cache value type. */
-    private String valTypeName;
-
-    /** Group to put new cache into. */
-    private String cacheGrp;
-
-    //    /** Atomicity mode for new cache. */
-    //    private CacheAtomicityMode atomicityMode;
-    //
-    //    /** Write sync mode. */
-    //    private CacheWriteSynchronizationMode writeSyncMode;
-
-    /** Backups number for new cache. */
-    private Integer backups;
-
+public class CreateTableCommand extends AbstractDdlCommand {
+    /** Replicas number. */
+    private Integer replicas;
```

Comment:
Should this be 'int' ?
What does 'null' value mean?

Reply:
null means default.